ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Get_Vendor_Files/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+##############
+# VENDOR_FILES
+##############
+
+- name: check out AdoptOpenJDK/secrets
+  local_action: git repo=git@github.com:AdoptOpenJDK/secrets.git dest=vendor_files
+  register: git_clone
```

Comment:
It doesn't look like you use the `git_clone` var?

Reply:
fixed